src/clearwater_modules/nsm1/dynamic_variables.py

```diff
@@ -0,0 +1,1408 @@
+    use='dynamic',
+    process=processes.mub_max_tc
+)
+
+Variable(
+    name='krb_tc',
+    long_name='Benthic algae respiration rate',
+    units='1/d',
+    description='Benthic algae respiration rate with temperature correction',
+    use='dynamic',
+    process=processes.krb_tc
+)
+
+Variable(
+    name='kdb_tc',
+    long_name='Benthic algae mortality rate',
+    units='1/d',
+    description='Benthic algae mortality rate with temperature correction',
+    use='dynamic',
+    process=processes.kdb_tc
+)
+
+Variable(
+    name='rnb',
+    long_name='Ratio nitrogen to dry weight',
+    units='mg-N/mg-D',
+    description='Ratio benthic algae nitrogen to dry weight',
+    use='dynamic',
+    process=processes.rnb
+)
+
+Variable(
+    name='rpb',
+    long_name='Ratio benthic algae phosphorus to dry weight',
+    units='mg-P/mg-D',
+    description='Ratio benthic algae phosphorus to dry weight',
+    use='dynamic',
+    process=processes.rpb
+)
+
+Variable(
+    name='rcb',
+    long_name='Ratio benthic algae carbon to dry weight',
+    units='mg-C/mg-D',
+    description='Ratio benthic algae carbon to dry weight',
+    use='dynamic',
+    process=processes.rcb
+)
+
+Variable(
+    name='rab',
+    long_name='Ratio benthic algae chlorophyll-a to dry weight',
+    units='ug-Chala-a/mg-D',
+    description='Ratio benthic algae chlorophyll-a to dry weight',
+    use='dynamic',
+    process=processes.rab
+)
+
+Variable(
+    name='FLb',
+    long_name='Benthic algal light limitation factor',
+    units='unitless',
+    description='Benthic algal light limitation factor',
+    use='dynamic',
+    process=processes.FLb
+)
+
+Variable(
+    name='FNb',
+    long_name='Benthic algal nitrogen limitation factor',
+    units='unitless',
+    description='Benthic algal nitrogen limitation factor',
+    use='dynamic',
+    process=processes.FNb
+)
+
+Variable(
+    name='FPb',
+    long_name='Benthic algal phosphorous limitation factor',
+    units='unitless',
+    description='Benthic algal phosphorous limitation factor',
+    use='dynamic',
+    process=processes.FPb
+)
+
+Variable(
+    name='FSb',
+    long_name='Benthic algal density attenuation',
+    units='unitless',
+    description='Benthic algal density attenuation',
+    use='dynamic',
+    process=processes.FSb
+)
+
+Variable(
+    name='mub',
+    long_name='Benthic algae specific growth rate',
+    units='1/d',
+    description='Benthic algae specific growth rate',
+    use='dynamic',
+    process=processes.mub
+)
+
+Variable(
+    name='AbGrowth',
+    long_name='Benthic algae growth rate',
+    units='g/m^2/d',
+    description='Benthic algae growth rate',
+    use='dynamic',
+    process=processes.AbGrowth
+)
+
+Variable(
+    name='AbRespiration',
+    long_name='Benthic algae respiration rate',
+    units='g/m^2/d',
+    description='Benthic algae respiration rate',
+    use='dynamic',
+    process=processes.AbRespiration
```

Comment:
I think mixed capitals/lower case also affects how the text is formatted in text editors (for example, here any function with mixed upper/lower case is formatted like a class rather than a function). Doesn't impact function, but is an indicator that something isn't aligned with best practice.

Reply:
I'm a big fan of renaming variables to align with BOTH Python style conventions AND CF standard names, IF we also keep track of previous names for cross-referencing to Fortran and previous docs.  

That said, I also agree that this should be done in a future effort and PR. Let's get this working first!

Reply:
Sounds good -- looks like this issue exists already #19. I added the nsm tag and we can address in the future.